bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -307,7 +307,10 @@ void ReplicaImp::tryToSendPrePrepareMsg(bool batchingLogic) {
     DebugStatistics::onSendPrePrepareMessage(pp->numberOfRequests(), requestsQueueOfPrimary.size());
   }
   primaryLastUsedSeqNum++;
-
+  {
+    concordlogger::MDC content(GL, "cid", "content: " + pp->getAllRequestsCorrelationIdAsString());
+    LOG_INFO(GL, "");
```

Comment:
Is that empty so that it just logs the content from above?

Reply:
Yes, and in the specific format of MDC such that it will be easy to parse it